### PR TITLE
Replace padding with PADDING Frame

### PR DESCRIPTION
**Pull request by gorryfair**
Changes to explain Padding == PADDING Frame

### CODE REVIEW

**Review by janaiyengar — APPROVED** [review]
I've done a bit of editorializing, and this LGTM. @martinthomson should take a look.

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -2963,13 +2964,14 @@ data (or 5 bytes, less the two fixed bits).
...
-sends to at least 22 bytes longer than the minimum connection ID that it might
-request the peer to include in packets that the peer sends.  This ensures that
-any stateless reset sent by the peer is indistinguishable from a valid packet
-sent to the endpoint.  An endpoint that sends a stateless reset in response to a
-packet that is 43 bytes or shorter SHOULD send a stateless reset that is one
-byte shorter than the packet it responds to.
+of a connection ID. To achieve that end, the endpoint SHOULD, adding PADDING
+frames as necessary, ensure that all packets it sends are at least 22 bytes
+longer than the minimum connection ID length that it requests the peer to
+include in its packets.  This ensures that any stateless reset sent by the peer
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md:2970]:
```suggestion
of a connection ID. To achieve that end, the endpoint SHOULD ensure that all
packets it sends are at least 22 bytes longer than the minimum connection ID
length that it requests the peer to include in its packets, adding PADDING
frames as necessary.  This ensures that any stateless reset sent by the peer
```

**Review by martinthomson — APPROVED** [review]
The commas for parentheticals got a little awkward there.  Otherwise, this is a good change.

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -1903,9 +1904,9 @@ that contain packets that are successfully processed and datagrams that contain
 packets that are all discarded.
 
 Clients MUST ensure that UDP datagrams containing Initial packets have UDP
-payloads of at least 1200 bytes, adding padding to packets in the datagram as
-necessary. A client that sends padded datagrams allows the server to send more
-data prior to completing address validation.
+payloads of at least 1200 bytes, adding PADDING frames to QUIC packets in the
+datagram as necessary. A client that sends padded datagrams allows the server to
```
Comment (ianswett) [inline draft-ietf-quic-transport.md:1908]:
```suggestion
payloads of at least 1200 bytes, adding PADDING frames as necessary.
A client that sends padded datagrams allows the server to
```

**Comment by martinthomson** [comment]
Thanks again Gorry.

**Pull request merged**